calcom/cal.com
# fix(api): Added correct include for `GET /teams/{teamId}/event-types`

## What does this PR do?

API for `GET /teams/{teamId}/event-types` does not include `hosts` and other fields that `GET /event-types` does.

Fixes # (issue)

https://github.com/calcom/cal.com/issues/12402

## Requirement/Documentation

> N/A

## Type of change




## How should this be tested?



- Are there environment variables that should be set? 
> :x: 
- What are the minimal test data to have?
> A team event type with 1 or more hosts (either fixed and/or RR will do)
- What is expected (happy path) to have (input and output)?
> API request to `GET /v1/teams/1/event-types` should have a correctly populated `hosts` array.
- Any other important info that could help to test that PR
> :x: 

## Mandatory Tasks


## Checklist



- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
